execute/report/report_test.go

```diff
@@ -158,25 +386,6 @@ func Test_buildSingleChainReport_Errors(t *testing.T) {
 				tokenDataReader: badTokenDataReader{},
 			},
 		},
-		{
-			name:    "bad codec",
-			wantErr: "unable to encode report: bad codec",
-			args: args{
-				report: plugintypes.ExecutePluginCommitDataWithMessages{
-					ExecutePluginCommitData: plugintypes.ExecutePluginCommitData{
-						SourceChain:         1234567,
-						SequenceNumberRange: cciptypes.NewSeqNumRange(cciptypes.SeqNum(100), cciptypes.SeqNum(100)),
-					},
-					Messages: []cciptypes.Message{
-						{Header: cciptypes.RampMessageHeader{
-							SourceChainSelector: 1234567,
-							SequenceNumber:      cciptypes.SeqNum(100),
-						}},
-					},
-				},
-				codec: badCodec{},
-			},
-		},
```

Comment:
This case will return when I test the new `checkMessage` and `verifyReport` methods.